platform/nativescript/plugins/navigator-plugin.js

```diff
@@ -89,6 +89,10 @@ export default {
 
         updateDevtools()
 
+        const resolveOnEvent = options.resolveOnEvent === true
```

Comment:
Wouldn't this make `resolveOnEvent` into a boolean? And then on line 112: 
```js
 page.on(resolveOnEvent, resolveHandler)
```
Would try to do 
```js
 page.on(true, resolveHandler)
```

```suggestion
        const resolveOnEvent = options.resolveOnEvent
```